config/locales/en/activerecord/course/assessment/answer/programming_file.yml

```diff
@@ -4,5 +4,7 @@ en:
       models:
         course/assessment/answer/programming_file:
           attributes:
+            filename:
+              incompatible_file: 'Incompatible file'
```

Comment:
Where is this used?

Reply:
sorry. this message was the first one I intended to use for denoting that the file is incompatible for Java programming import files. However, there's already the one with more specific error messages, and hence this one is unused anymore and hence I deleted it.